src/main/java/toast/specialMobs/entity/witch/Entity_SpecialWitch.java

```diff
@@ -394,6 +403,13 @@ protected void dropFewItems(boolean hit, int looting) {
         }
     }
 
+    @Optional.Method(modid = "mobsinfo")
+    @Override
+    public void provideDropsInformation(@Nonnull ArrayList<MobDrop> drops) {
+        IMobInfoProvider.provideSuperVanillaDrops(drops, EntityWitch.class);
+        // TODO: what to do with drinking potion ????
```

Comment:
I think there should be added all potions mentioned in `tryDrinkPotion` method with chance of 15% (as stated in `dropFewItems`). At least what I can say looking at the code. The better approach would probably be to verify these values by play testing.

Reply:
Will look at it later, it has to be fixed on normal witch drops as well